x/rollapp/keeper/invariants.go

```diff
@@ -291,17 +291,28 @@ func LivenessEventInvariant(k Keeper) sdk.Invariant {
 				msg += fmt.Sprintf("| rollapp stored event but wrong number found in queue: rollapp: %s: event height: %d: found: %d", ra.RollappId, ra.LivenessEventHeight, cnt)
 			}
 		}
-		for _, e := range k.GetLivenessEvents(ctx, nil) {
+		evts := k.GetLivenessEvents(ctx, nil)
+		seen := make(map[string]struct{})
+		for i, e := range evts {
+			if 0 < i && e.HubHeight < evts[i-1].HubHeight {
+				broken = true
+				msg += fmt.Sprintf("| events not sorted by height: event: %v\n", e)
```

Comment:
is it guaranteed to be sorted by height? is it even an invariant?
it has no effect on the app

Reply:
It is, there is a query to find in range